Description: Revise cleanup to remove all generated files.
 The qperf build uses the script "cleanup" to remove build output. This
 patches cleanup to remove files that the upstream version leaves behind.
Author: Brian T. Smith <bsmith@systemfabricworks.com>
Bug: https://github.com/linux-rdma/qperf/issues/4
Last-Update: 2017-11-16
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/cleanup
+++ b/cleanup
@@ -22,6 +22,7 @@
     src/help.c
     src/qperf.1
     src/qperf
+    compile
 EOF`
 
 # We need to keep qperf.spec around for the OFED scripts but delete it if we do
